src/modules/handleCompleted.js

```diff
@@ -0,0 +1,11 @@
+import { getTask } from './storeTask.js';
+
+const handleCompleted = (index) => {
+  const completed = document.getElementById(`check${index}`).toggleAttribute('checked');
+  const tasks = getTask();
+
+  tasks[index].completed = completed;
+  localStorage.setItem('data', JSON.stringify(tasks));
+};
```

Comment:
According to the project requirements, we should update the task object's values for completed `Items` upon user actions.

However, In your app, when I marked `Item One` and `Item Two` as completed, different items were updated 👀, therefore that is affecting the result of the "Clear all completed" button.

![image](https://github.com/over-geek/Todo-List/assets/78251927/1d59c380-88ca-49f6-a180-fdc19d5677f4)

Kindly fix this issue. 🛠😊